main.go

```diff
@@ -83,16 +86,17 @@ func main() {
 		Namespace:          namespace,
 	})
 	if err != nil {
-		setupLog.Error(err, "unable to start manager")
+		setupLog.Error(err, "Unable to start manager")
 		os.Exit(1)
 	}
 
 	if err = (&controllers.ManagedOCSReconciler{
-		Client: mgr.GetClient(),
-		Log:    ctrl.Log.WithName("controllers").WithName("ManagedOCS"),
-		Scheme: mgr.GetScheme(),
+		Client:               mgr.GetClient(),
+		Log:                  ctrl.Log.WithName("controllers").WithName("ManagedOCS"),
+		Scheme:               mgr.GetScheme(),
+		AddonParamSecretName: getAddonParamSecretName(),
```

Comment:
I don't see a value in a helper for a single call site, especially where the code inside the helper is less than the helper itself. 
Doing it just forces the reader to jump elsewhere in the file to understand the code which is, in my opinion,  cumbersome and annoying. 

```suggestion
		AddonParamSecretName: fmt.Sprintf("addon-%v-parameters", envVars[addonNameEnvVarName])
```